### PR TITLE
Use default client in function

### PR DESCRIPTION
**Pull request by WilliamBergamin**
### Type of change (place an x in the [ ] that applies)

- [ ] New sample
- [ ] New feature
- [x] Bug fix
- [ ] Documentation

### Summary

This PR updates the project to use the default `client` ver using the `token` and creating a new client, I also took the opportunity to upgrade to `SlackFunction` since SlackFunctionHandler is deprecated.

### Requirements (place an x in each [ ] that applies)

- [x] I’ve checked my submission against the Samples Checklist to ensure it complies with all standards
- [x] I have ensured the changes I am contributing align with existing patterns and have tested and linted my code
- [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct)

### CODE REVIEW

File: functions/create_welcome_message.ts [hunk 1]
```diff
@@ -33,55 +32,50 @@ export const WelcomeMessageSetupFunction = DefineFunction({
...
+            value: "{{data.channel_id}}",
+          },
+          triggered_user: {
+            value: "{{data.user_id}}",
+          },
+        },
+      });
 
-export default setupFunction;
+      console.log(triggerResponse);
```
Comment (misscoded) [inline functions/create_welcome_message.ts:75]:
```suggestion
```

File: functions/create_welcome_message.ts [hunk 2]
```diff
@@ -33,55 +32,50 @@ export const WelcomeMessageSetupFunction = DefineFunction({
...
-    };
-  }
-};
+    if (!putResponse.ok) {
+      return await {
+        error: putResponse.error,
+        outputs: {},
+      };
+    } else {
+      const triggerResponse = await client.workflows.triggers.create({
```
Comment (misscoded) [inline functions/create_welcome_message.ts:56]:
```suggestion
      await client.workflows.triggers.create({
```

File: functions/create_welcome_message.ts [hunk 3]
```diff
@@ -33,55 +32,50 @@ export const WelcomeMessageSetupFunction = DefineFunction({
...
+          },
+          triggered_user: {
+            value: "{{data.user_id}}",
+          },
+        },
+      });
 
-export default setupFunction;
+      console.log(triggerResponse);
+      return await {
```
Comment (misscoded) [inline functions/create_welcome_message.ts:76]:
```suggestion
      return {
```

File: functions/send_welcome_message.ts [hunk 4]
```diff
@@ -53,6 +49,4 @@ const setupFunction: SlackFunctionHandler<
   return await {
```
Comment (misscoded) [inline functions/send_welcome_message.ts:49]:
```suggestion
  return {
```
Unless there's something I'm not understanding about the code as written, `await` shouldn't be necessary? 🤔

**Review by misscoded — APPROVED** [review]
Thanks for making these changes! 🙇🏼‍♀️ 

All comments left are nits, save for the removal of the rogue `console.log` (unless that was intentional)

**Comment by WilliamBergamin** [comment]
Thanks for the feedback @misscoded 🥇 Ive implemented the changes

I saw a few areas of improvements for this sample, notably adding unit tests. I opted to only include the changes targeted to my current task in this PR, leave the other improvements for another Issue + PR do you know if there already exists an Issue about this?

**Pull request merged**